packages/cyberstorm/src/components/componentStyles/MenuItem.module.css

```diff
@@ -18,25 +18,29 @@
   --bg-color: transparent;
 }
 
+.root[data-highlighted] {
+  z-index: 999;
+}
```

Comment:
I'm not sure if it's best to include the data attribute from Radix here in a component that doesn't rely on Radix at all and could potentially used without any involvement from Radix. Would this be better suited to reside in `DropDown` and `Select`?

Reply:
Created a clickup task about this

Reply:
Fixed